app/src/androidTest/java/br/com/concrete/tentacle/features/library/loan/LoanActivityTest.kt

```diff
@@ -0,0 +1,98 @@
+package br.com.concrete.tentacle.features.library.loan
+
+import android.content.Intent
+import androidx.fragment.app.Fragment
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import br.com.concrete.tentacle.R
+import br.com.concrete.tentacle.extensions.getJson
+import br.com.concrete.tentacle.features.library.LoanActivity
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.hamcrest.Matchers
+import org.hamcrest.core.IsNot.not
```

Comment:
Em um lugar está sento utilizado o "not" do "org.hamcrest.core.IsNot.not" e em outros o "Matchers.not", acho melhor usar somente um dos dois!